src/test/kotlin/com/ktor/solution/ConfigureRouting.kt

```diff
@@ -0,0 +1,136 @@
+package com.ktor.solution
+
+import io.ktor.client.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+import io.ktor.server.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.server.testing.*
+import kotlinx.serialization.json.Json
+
+val emtyPost = mutableMapOf<Int, Post>()
+val posts = mutableMapOf<Int, Post>(
+    1 to Post(userId = 1, id = 1, title = "Original Post", body = "Original body.")
+)
+
+fun Application.configureRouting() {
```

Comment:
ConfigureRouting is not appropriate name for class, it looks like a function. Router is better (as a simplest option)